webapp/app/components/errors/traceback-frame/component.scss

```diff
@@ -10,7 +10,7 @@
         visibility: visible;
     }
 
-    &.test-code .frame-code {
+    .test-code, .frame-code {
```

Comment:
I don't think the comma is correct - this only apples to a `.frame-code` directly beneath a `.test-code`...